packages/pwa/app/pages/home/index.jsx

```diff
@@ -191,17 +243,38 @@ const Home = () => {
 }
 
 Home.getTemplateName = () => 'home'
-Home.propTypes = {
-    recommendations: PropTypes.array,
-    isLoading: PropTypes.bool
-}
 
-Home.getProps = async ({res}) => {
+Home.shouldGetProps = ({previousLocation, location}) =>
+    !previousLocation || previousLocation.pathname !== location.pathname
+
+Home.getProps = async ({res, api}) => {
     // Since the home page is static, it is safe to set max age to a high value
     // we set it to a year here, but you can set the value that is suitable for your project
     if (res) {
         res.set('Cache-Control', 'max-age=31536000')
     }
+
+    const productSearchResult = await api.shopperSearch.productSearch({
+        parameters: {
+            refine: [`cgid=${SHOP_PRODUCTS_CATEGORY_ID}`, 'htype=master'],
+            limit: SHOP_PRODUCTS_LIMIT
+        }
+    })
```

Comment:
For this PR, I agree that opting to hardcode this category id makes for a simpler homepage.

Topic for a future ticket/PR: 
But this brings to light the question of what to do if we want to fetch categories (e.g. to look up the id of the first category) in other pages' `getProps`. Like Adam said, we would want to avoid making duplicate calls to the categories endpoint. In other words, how do we deduplicate requests like that?

Perhaps we would need some logic in our Commerce API client that:
- caches duplicate requests
- manages the async calls ← in our scenario, App and the page-level component would attempt to fetch the categories at the same time

Perhaps instead of reinventing the wheel, we can explore using `react-query` library too.

Reply:
I agree. This is probable a larger topic revolving around global data that we have purposefully avoided in the past. I think it's something we'll want to take a look at when the time is right and we aren't working on more important things. A natural time to look into this might be when we are looking at refactoring the commerce api hooks.